tests/README.md

```diff
@@ -59,52 +58,51 @@ To use an alternate version, simply set this variable before running the relevan
 For example, if you wanted to use `v1.9.6`, you would run `export KUBE_VERSION=v1.9.6` first before running `up`.
 
 ### Install Helm
-Use [helm.sh](/tests/scripts/helm.sh) to install helm and set up rook charts defined under `_output/charts` (generated by build).
- - To install and set up Helm charts for rook run `tests/scripts/helm.sh up` 
- - To clean up `tests/scripts/helm.sh clean`
+Use [helm.sh](/tests/scripts/helm.sh) to install helm and set up Rook charts defined under `_output/charts` (generated by build):
+ - To install and set up Helm charts for Rook run `tests/scripts/helm.sh up`.
+ - To clean up `tests/scripts/helm.sh clean`.
 
-**Note:**  *kubeadm.sh, minikube.sh and helm.sh scripts depend on some artifacts under _output dir generated during build time, these scripts 
-should be run from project root. e.g., `tests/script/kubeadm.sh up`
+**Note:** `*kubeadm.sh`, `minikube.sh` and `helm.sh` scripts depend on some artifacts under the `_output/` directory generated during build time,
+these scripts should be run from project root. e.g., `tests/script/kubeadm.sh up`.
 
 ## Run Tests
 From the project root do the following:
-#### 1. Build rook: 
- run `make build`
-
-#### 2. Start kubernetes 
-using one of the following:
-   
-   -  using Kubeadm  
-   ```
-    tests/scripts/kubeadm.sh up
-    tests/scripts/helm.sh up  
-  ```   
-   - using minikube
-   ```
-   tests/scripts/minikube.sh up
-   tests/scripts/minikube.sh helm
-   tests/scripts/helm.sh up
-   ```
-
-#### 3. Run integration tests: 
-Integration tests can be run using tests binary `_output/tests/${platform}/integration` that is generated during build time
-eg ` ~/integration -test.v `
-       
+#### 1. Build rook:
+Run `make build`
 
-### Test parameters
-In addition to standard go tests parameters, the following custom parameters are available while running tests
+#### 2. Start Kubernetes
+Using one of the following:
 
- Parameter | Description | Possible values | Default
- --- |--- | --- | ---
-rook_platform| platform rook needs to be installed on  | kubernetes | kubernetes
-k8s_version  | version of Kubernetes to be installed  | v1.7+  | v1.8.5
-rook_image | rook image name to be installed | valid image name | rook/ceph
-toolbox_image | toolbox image name to be installed | valid image name | rook/ceph-toolbox
-skip_install_rook | skips installing rook (if already installed) | true or false  | false
+-  using Kubeadm  
+```
+tests/scripts/kubeadm.sh up
+tests/scripts/helm.sh up  
+```
+- using minikube
+```
+tests/scripts/minikube.sh up
+tests/scripts/minikube.sh helm
+tests/scripts/helm.sh up
+```
 
-### Running Tests with parameters.
+#### 3. Run integration tests:
+Integration tests can be run using tests binary `_output/tests/${platform}/integration` that is generated during build time e.g.:
+```
+~/integration -test.v
+```
 
-#### To run all integration tests run 
+### Test parameters
```

Comment:
This table is broken in the rendering on https://github.com/galexrt/rook/blob/583d156e5602a8d0390a15bc0925f1b93d03352a/tests/README.md#test-parameters.  You probably want to check all the other tables that were touched in this PR to ensure no others were broken.

See the original rendering that isn't broken here: https://github.com/rook/rook/blob/master/tests/README.md#test-parameters